tests/casp/stack_controller.pm

```diff
@@ -74,7 +74,8 @@ sub velum_bootstrap {
     if (check_screen('velum-bootstrap-accept-nodes', 3)) {
         assert_and_click 'velum-bootstrap-accept-nodes';
         # Wait until nodes are moved from pending
-        assert_screen 'velum-nodes-accepted';
+        my $nodes = get_var('STACK_SIZE') - 2;
```

Comment:
I understand that this is probably obvious for you , but anyway I think comment explaining this will not harm